framework/src/controller/schema/application_config_schema.js

```diff
@@ -56,6 +61,15 @@ module.exports = {
 					example: '968d7b5b97a5bfad8f77614dc8a9918de49f6c6e',
 					description: 'The version of Lisk Core that the peer node runs on.',
 				},
+				tempPath: {
+					type: 'string',
+					format: 'path',
+					minLength: 1,
+					maxLength: 50,
```

Comment:
Can you add a comment here explaining why we have max length limit, bitte?

Reply:
Ok, will add a description for this and expanded description with the calculations on the issue itself, thanks! 

`Restrictions due to unix domain socket path length`.